### PR TITLE
Clean up: local file

### PR DESCRIPTION
**Pull request by UlrikeS91**
MINOR changes:
- improved instructions 
- establish alphabetical order 
- fixed indentation

MAJOR changes:
none

SPECIAL ATTENTION: 
none

### CODE REVIEW

File: schemas/localFile.schema.tpl.json [hunk 1]
```diff
@@ -1,58 +1,58 @@
...
+  "properties": {
+    "contentDescription": {
+      "type": "string",
+      "_instruction": "Enter a short content description for this local file instance."
+    },
+    "dataType": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add all data types that are specifially represented in this local file instance.",
```
Comment (apdavison) [inline schemas/localFile.schema.tpl.json]: spelling mistake "specifially" --> "specifically"

File: schemas/localFile.schema.tpl.json [hunk 2]
```diff
@@ -1,58 +1,58 @@
...
+      "_instruction": "Enter the file system path (absolute path or relative to the working directory) to this local file instance."
+    },
+    "specialUsageRole": {
+      "_instruction": "Add the special usage role of this local file instance.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/controlledTerms/FileUsageRole"
+      ]
+    },
+    "storageSize": {
+      "_instruction": "Enter the storage size this local file instance allocates.",
```
Comment (apdavison) [inline schemas/localFile.schema.tpl.json]: I'm not sure "allocates" is accurate. I suggest:

"Enter the storage size of this local file instance."

**Review by apdavison — CHANGES_REQUESTED** [review]

File: schemas/localFile.schema.tpl.json (hunk 1 above)
Comment (UlrikeS91) [reply schemas/localFile.schema.tpl.json]: Thanks for spotting this!

File: schemas/localFile.schema.tpl.json (hunk 2 above)
Comment (UlrikeS91) [reply schemas/localFile.schema.tpl.json]: Agreed. Will change it to your suggestion.

**Review by lzehl — APPROVED** [review]

**Comment by lzehl** [comment]
@apdavison feel free if you are happy with the changes

**Pull request merged**